visualizations/frontend/other/TTable.vue

```diff
@@ -364,6 +365,10 @@ export default {
       type: Array,
       default: () => [],
     },
+    extraClass: {
+      type: String,
+      default: "",
+    },
```

Comment:
While this works, the table is so vast, it could be very difficult to know what `extraClass` applies to. 

We might want to call this `borderClasses` for now, so it's clear what these styles target.

Reply:
I am using this single class for multiple things. Like `border border-[#D3D3D9] !overflow-hidden !m-0 !mt-4 rounded-md`

Reply:
We can call this class `multiPurpose`.